src/components/Home/Balance.js

```diff
@@ -31,6 +31,14 @@ export default class Balance extends React.PureComponent {
 						{t('no_balance')}
 					</Text>
 				)}
+				<Text style={{ fontSize: 13, color: colors.secondary }}>
+					{thisWeek ? (
+						<Text style={{ fontWeight: 'bold', color: colors.less }}>{thisWeek}</Text>
+					) : (
+						_('loading_text_replacement')
+					)}{' '}
+					{t('this_week')}
```

Comment:
"0,00€ dépensés" here a problem, should be "dépensé" even for < 1 and if i have refilled, i got +20€ so you need to change the expression

Reply:
Nice found

Reply:
Will be fixed by another PR